conanfile.py

```diff
@@ -16,12 +16,14 @@ class EmbeddedPython(ConanFile):
         "packages": "ANY", 
         "pip_version": "ANY",
         "pip_licenses_version": "ANY",
+        "setuptools_version": "ANY",
         "openssl_variant": ["lowercase", "uppercase"]  # see explanation in `build_requirements()`
     }
     default_options = {
         "packages": None,
         "pip_version": "21.2.4",
         "pip_licenses_version": "3.5.3",
+        "setuptools_version": "53.0.0",
```

Comment:
Can you check if a newer version of `setuptools` will work as the default? This one's ancient: https://pypi.org/project/setuptools

Reply:
The latest our project can support right now is `57.5.0`, I could set that as the default. But I could also set the latest `58.5.3`. Any preference?

Reply:
Go for 57.5.0 if that works for everyone.

Reply:
Done in the latest fxup.